WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController.swift

```diff
@@ -975,11 +975,13 @@ extension GutenbergViewController: GutenbergBridgeDataSource {
     }
 
     func gutenbergCapabilities() -> [Capabilities: Bool] {
+        let isFreeWPCom = post.blog.isHostedAtWPcom && !post.blog.hasPaidPlan
         return [
             .mentions: FeatureFlag.gutenbergMentions.enabled && SuggestionService.shared.shouldShowSuggestions(for: post.blog),
             .xposts: FeatureFlag.gutenbergXposts.enabled && SiteSuggestionService.shared.shouldShowSuggestions(for: post.blog),
             .unsupportedBlockEditor: isUnsupportedBlockEditorEnabled,
             .canEnableUnsupportedBlockEditor: post.blog.jetpack?.isConnected ?? false,
+            .audioBlock: !isFreeWPCom // Disable audio block until it's usable on free sites via "Insert from URL" capability
```

Comment:
Just a note here, so we would keep this flag but rename it to something like "canEnableAudioBlockMediaUpload" since at the point of the merge of the change from URL feature, those features should still be disabled.

Reply:
Another thought, I am going to make the logic on Android identical to this for clarity. I will keep this comment as well 😄

Reply:
Yeah I think too that `let isFreeWPCom = post.blog.isHostedAtWPcom && !post.blog.hasPaidPlan` reflects the purpose better than `let canEnableAudioBlockMediaUpload = !post.blog.isHostedAtWPcom || post.blog.hasPaidPlan`